1-JavaScript/Week4/homework/ex3-lemonAllergy.js

```diff
@@ -23,8 +23,11 @@ const fruitBasket = [
   'lemon',
 ];
 
-function sanitizeFruitBasket(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+
+function sanitizeFruitBasket(arr,takenOut) {
+const list=arr.filter(out => out !== takenOut)
+
+return `My mom bought me a fruit basket containing ${list}!`
```

Comment:
good job!

one detail: you can see as in the example the fruits are separated by a comma and one space, could you try to achieve the same result?

Hint: the method Array.join can help you to convert an array to a string. The first argument of this method expects a separator.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join